src/norm/private/postgres/dbtypes.nim

```diff
@@ -67,9 +67,9 @@ func to*(dbVal; T: typedesc[SomeFloat]): T = dbVal.f.T
 
 func to*(dbVal; T: typedesc[string]): T = dbVal.s
 
-func to*[_](dbVal; T: typedesc[StringOfCap[_]]): T = dbVal.o.value.T
+func to*[U](dbVal; T: typedesc[StringOfCap[U]]): T = dbVal.o.value.T
 
-func to*[_](dbVal; T: typedesc[PaddedStringOfCap[_]]): T = dbVal.o.value.T
+func to*[U](dbVal; T: typedesc[PaddedStringOfCap[U]]): T = dbVal.o.value.T
```

Comment:
I think U is confusing since it doesn't mean anything.

How about T1 and T2?

Reply:
Works for me, Done